webpack/config.js

```diff
@@ -22,6 +21,11 @@ const jsxLoaders = [{ loader: "babel-loader" }];
 const assetsDir = process.env.ASSETS_DIR;
 const assetMapFile = process.env.ASSETS_MAP_FILE;
 const outputFile = DEBUG ? "[name].js" : "[name].[chunkhash].js";
+console.log("Webpack starting with config", {
+  assetsDir,
+  assetMapFile,
+  outputFile
+});
```

Comment:
It was helpful to see this when sorting out config in different environments. I can delete if it seems overly verbose.